### PR TITLE
homework2: Sergei Tishkevich

### PR DESCRIPTION
**Pull request by ghost**
homework2.

### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle)
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks
- [ ] my PR made from the fork feature branch to maxim homework# branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

**Comment by ghost** [comment]
homework2.

File: src/homework2/task3.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    :param str_: входная строка
+    :return: строка. Получившееся выражение
+    """
+
+    # write your code here
+    return ''  # write return value here
+    s = input('Введите предложение: ')
+    str_=''
+    for i in s:
+        if i not in str_ and i!=' ': str_+=i
```
Comment (belovmd) [inline src/homework2/task3.py:21]: please move operation to next line

File: src/homework2/task4.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    :param str_: входная строка
+    :return: кортеж. (low_number, up_number). low_number - количество строчных,
+                                              up_number - количество пописных.
+    """
+
+    # write your code here
+
+
+s = input()
```
Comment (belovmd) [inline src/homework2/task4.py:17]: please put your solution in function. in fact this file wouldn't be run because of indentation error

**Review by belovmd — COMMENTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-summer/blob/master/Homework%20review/Homework2.md):
- 10
- missed tasks descriptions in all external sites tasks
- 12
- 16
- 18

File: src/homework2/task6.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    n = input("введите число : ")
+
+
+rev = 0;
+l = len(n);
+
+m = int(n);
+
+for i in range(0, l):
```
Comment (belovmd) [inline src/homework2/task6.py:25]: not in func

**Comment by belovmd** [comment]
wrong branches - no ci checks (they will fail)
no advanced tasks
not addressed comments for task 2
totally too unaccurate
5,5

**Pull request closed without merging**